about.html

```diff
@@ -141,75 +141,75 @@ <h2 class="content-subhead">Controls and features</h2>
+                              <td aria-label="Unavailable"></td>
+                              <td aria-label="Unavailable"></td>
+                              <td aria-label="Unavailable"></td>
                             </tr>
                             <tr>
                               <th scope="row">Native C/C++</th>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
-                              <td> </td>
-                              <td> </td>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Unavailable"> </td>
+                              <td aria-label="Unavailable"> </td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
                             </tr>
                             <tr>
                               <th scope="row">.NET 5 Support</th>
-                              <td><i class="icon-check text-icon"></i></td>
-                              <td> </td>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
-                              <td> </td>
+                              <td aria-label="Available"><i class="icon-check text-icon"></i></td>
+                              <td aria-label="Unavailable"> </td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Unavailable"> </td>
                             </tr>
                             <tr>
                               <th scope="row">WebView2<br/>(Chromium-based engine)</th>
-                              <td><i class="icon-check m-auto text-icon"></i></td>
-                              <td> </td>
-                              <td><i class="icon-check text-icon" style="margin-left:18px;"></i> **</td>
-                              <td><i class="icon-check text-icon" style="margin-left:18px;"></i> **</td>
-                              <td> </td>
+                              <td aria-label="Available"><i class="icon-check m-auto text-icon"></i></td>
+                              <td aria-label="Unavailable"> </td>
+                              <td aria-label="Available as work in progress"><i class="icon-check text-icon" style="margin-left:18px;"></i> **</td>
```

Comment:
Can we change this to: "Currently a work in progress"?

Reply:
Of course @anawishnoff, updated the wording now!